gnssanalysis/gn_io/igslog.py

```diff
@@ -101,6 +129,22 @@ def find_recent_logs(logs_glob_path: str, rnx_glob_path=None) -> _pd.DataFrame:
     return recent_logs_df
 
 
+def determine_log_version(data: bytes):
+    REGEX_VERSION_1 = _re.compile(rb"""(site log\))""")
+    REGEX_VERSION_2 = _re.compile(rb"""(site log v2)""")
+
+    # check for version 1:
+    result_v1 = REGEX_VERSION_1.search(data)
+    result_v2 = REGEX_VERSION_2.search(data)
+
+    if result_v1 is not None:
```

Comment:
`if result_v1 is not None` should have the same effect then `if result_v1`

Reply:
Changed to `if result_v1`